src/jetstream/main.go

```diff
@@ -123,6 +132,13 @@ func main() {
 	// Create common method for looking up config
 	envLookup := getEnvironmentLookup()
 
+	// Update log output given all env vars
```

Comment:
I think we should move envLookup := getEnvironmentLookup() to the top and only check once - as is, you can get mixed text and json logging in the output, if you set the flag in a config file rather than an env var.

Need to check the getEnvironmentLookup function, but I think it should be okay to move up.